work/sixth/index.html

```diff
@@ -15,7 +15,59 @@
   </div>
   <script src="../../common/js/jquery.js"></script>
   <script>
-    // 回答はここに書いてね
+    $(function () {
+      let id = Math.floor(Math.random() * 151) + 1;
+      let url = `https://pokeapi.co/api/v2/pokemon-species/${id}`;
+      let encountHtml =
+      `
+      <p class="name"></p>
+      <div>
+        <img class="img" src="https://raw.githubusercontent.com/PokeAPI/sprites/master/sprites/pokemon/${id}.png" alt="">
+      </div>
+      <div>
+        <button class="btn" id="catch">捕まえる</button>
+        <button class="btn" data-js="reset">逃げる</button>
+      </div>
+      `;
+      let doneHtml =
+      `
+      <p class="name">ゲットしました</p>
+      <div>
+        <img class="img" src="https://raw.githubusercontent.com/PokeAPI/sprites/master/sprites/items/poke-ball.png" alt="">
+      </div>
+      <button class="btn" data-js="reset">戻る</button>
+      `;
+      let failHtml =
+      `
+      <p class="text">逃げられてしまった...</p>
+      <button class="btn mt" data-js="reset">戻る</button>
+      `;
+
+      $('#pokemon_search').click(function () {
+        $.ajax({
+          url: url,
```

Comment:
プロパティのkeyの値とvalueの変数名が同じ場合、省略記法が使えます！
```suggestion
          url,
```

Reply:
[dcdf569](https://github.com/shoo611/jquery_curriculum_new-master/pull/1/commits/dcdf569114740cd341b2e21841bebecd20bd7bff)

教えていただきありがとうございます！
そのように修正いたしました！